Tasks/quotes.js

```diff
@@ -1,30 +1,26 @@
+'use strict';
 // Change double quotation to open or close quotation
 
-const EMPTY = '';
+const quotes = (str) => {
```

Comment:
Will be better to remove parentheses around `str`

Reply:
Our `.eslintrc.json` file uses the option 
```json
"arrow-parens": [
      "error",
      "always"
    ],
```
So we need to keep it in one style